http_service/tests/test_bug_classification.py

```diff
@@ -159,6 +159,35 @@ def do_request():
     }
 
 
+def test_for_missing_bugs(client, responses):
+    existed_bug_ids = ["1602463", "1619699"]
+    missing_bug_ids = ["1598744", "1615281", "1566486"]
+    all_bug_ids = [*existed_bug_ids, *missing_bug_ids]
+
+    change_time = str(time.time())
+
+    responses.add(
+        responses.GET,
+        "https://bugzilla.mozilla.org/rest/bug?id=1566486,1598744,1602463,1615281,1619699&include_fields=id&include_fields=last_change_time",
+        status=200,
+        json={
+            "bugs": [
+                {"id": bug_id, "last_change_time": change_time}
+                for bug_id in existed_bug_ids
+            ],
+        },
+    )
+
+    rv = client.post(
+        "/component/predict/batch",
+        data=json.dumps({"bugs": [int(bug_id) for bug_id in all_bug_ids]}),
+        headers={API_TOKEN: "test"},
+    )
+    assert rv.status_code == 202
+    bugs = retrieve_compressed_reponse(rv)["bugs"]
+    assert bugs.keys() == set(all_bug_ids), "All the queried bug IDs must be returned"
```

Comment:
nit: I would have the bug IDs as integers and covert it here to strings:
```py
set(map(str, all_bug_ids))
```

Reply:
Ok sir, I will correct it